hack/ci/windows.ps1

```diff
@@ -864,10 +903,13 @@ Try {
                 if (!($env:INTEGRATION_TESTFLAGS)) {
                     $env:INTEGRATION_TESTFLAGS = "-test.v"
                 }
-                Set-Location "$env:SOURCES_DRIVE`:\$env:SOURCES_SUBDIR\src\github.com\docker\docker"
+                #Set-Location "$env:SOURCES_DRIVE`:\$env:SOURCES_SUBDIR\src\github.com\docker\docker"
```

Comment:
Why was this commented-out? Is it no longer needed? (if so, we should remove it)

Reply:
Done